src/components/Header/Header.tsx

```diff
@@ -66,26 +83,40 @@ function HeaderItem({ label, pane }: { label: string; pane: ActivePane }) {
 export default function Header() {
   const classes = useStyles();
   const { state } = useAppStateContext();
+  const theme = useTheme();
+  const isTablet = useMediaQuery(theme.breakpoints.only('md'));
 
   const numberOfPanes = Object.keys(ActivePane).length / 2;
 
   return (
-    <div className={classes.header}>
-      <Container>
-        <Grid container alignItems="center" justifyContent="space-between" style={{ height: '100%' }}>
-          <HeaderItem pane={ActivePane.DeviceCheck} label="Device & Software Setup" />
-          <HeaderItem pane={ActivePane.Connectivity} label="Connectivity" />
-          <HeaderItem pane={ActivePane.Quality} label="Quality & Performance" />
-          <HeaderItem pane={ActivePane.Results} label="Get Results" />
-        </Grid>
-      </Container>
+    <>
+      <div className={classes.header}>
+        <Container>
+          <Grid
+            container
+            alignItems="center"
+            justifyContent={isTablet ? 'space-around' : 'space-between'}
+            style={{ height: '100%' }}
+            wrap={isTablet ? 'nowrap' : 'wrap'}
```

Comment:
nit: Wondering if we should put all this in a CSS class at this point. We're using lots of CSS in this file already. Might as well keep all the styling code in one spot, unless it needs a JS variable like like 112. Also there's less JS to test if we make CSS do all the work.